zerver/tests/test_messages.py

```diff
@@ -1232,34 +1232,68 @@ def test_edit_message_no_content(self):
     def test_edit_message_history(self):
         # type: () -> None
         self.login(self.example_email("hamlet"))
-        msg_id = self.send_message(self.example_email("hamlet"), "Scotland", Recipient.STREAM,
-                                   subject="editing", content="content before edit")
-        new_content = 'content after edit'
 
-        result = self.client_patch("/json/messages/" + str(msg_id), {
-            'message_id': msg_id, 'content': new_content
+        # Single-line edit
+        msg_id_1 = self.send_message(self.example_email("hamlet"), "Scotland", Recipient.STREAM,
+                                     subject="editing", content="content before edit")
```

Comment:
Since it has been already wrapped to new line, maybe format it to fit in 80 chars per line criteria.